script.skin.helper.colorpicker/addon.xml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+	<addon id="script.skin.helper.colorpicker" name="Skin Helper Service ColorPicker" version="1.0.0" provider-name="marcelveldt">
+	<requires>
+		<import addon="xbmc.python" version="2.13.0"/>
+		<import addon="xbmc.addon" version="12.0.0"/>
+		<import addon="script.module.pil" version="1.1.7"/>
```

Comment:
Well if you think that PIL actually is supported on Android, I can drop the image files all together so they can just be built on demand if needed, that would be a great advantage. I remember from like 2 years ago that the PIL package wasn't working on Android devices. 

The reason I use the image files for the colors in the first place because there doesn't seem to be a method to set the colordiffuse tag with an info property inside the itemlayout. Otherwise I could have filled the list with just a white image and the colordiffuse and I wouldn't be needing PIL and images at all.